docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md

```diff
@@ -1,27 +1,27 @@
 ---
 sidebar_position: 6
-title: AVS Holesky Dashboard Onboarding
+title: AVS Dashboard Onboarding
 ---
 
 
-This document defines interfaces that AVSs must implement to index their data for the V1 AVS Marketplace
+This document defines interfaces that AVSs must implement for us to be able to index their data for the V1 [AVS Marketplace](https://app.eigenlayer.xyz/avs).
 
 
 ## Interface
 
 ```javascript
 interface IServiceManager {
-// These 3 functions are just proxies to the same-named functions in the AVSDirectory
+// Below 3 functions are just proxies to the same-named functions in the AVSDirectory
 function registerOperatorToAVS(address operator, Signature memory signature);
 
 function deregisterOperatorFromAVS(address operator);
 
 function updateAVSMetadataURI(string calldata metadataURI);
 	
-	// Implement in ServiceManager
-	function getOperatorRestakedStrategies(address operator) returns (address[] memory)
+// Below 2 functions are needed for the frontend and payments MVP
```

Comment:
Added mention of payments MVP here. Not sure if we want to wait until it's actually live before having it on the official docs. But we also have to remember to add it as soon as we launch payments if we don't have it now.

Reply:
@samlaf yes - if possible, I'd love for us to hold off on that content until the MVP is live. Thank you

Reply:
[67f14c7](https://github.com/Layr-Labs/eigenlayer-docs/pull/208/commits/67f14c7ac2537ac5fe228db009bef11bc4995c8c)